spec/solitaire_spec.rb

```diff
@@ -1,7 +1,29 @@
 require 'spec_helper'
 require_relative '../solitaire'
 
-describe Solitaire, '#decrypt_message' do
+describe Solitaire, '#prepare(message)' do
+  it "discards non A-Z letters, groups in 5's, adds X's" do
+    message = 'Code in Ruby, live longer!'
+    expected_output = ["CODEI", "NRUBY", "LIVEL", "ONGER"]
+
+    prepared_message  = Solitaire.new.prepare(message)
```

Comment:
It's a bit more canonical to pass the `message` in the constructor as in `Solitaire.new(message).prepare`.